src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -55,6 +54,23 @@ function formatHeader(appointment) {
   }
 }
 
+function formatInstructions(instructions) {
+  if (!instructions) {
+    return null;
+  }
+
+  return instructions.split('\n').reduce((obj, str) => {
```

Comment:
It looks like this splitting on line breaks logic is new, what's the reason for it?

Reply:
Yeah I don't know why I did that in the end, I just adjusted it now.